src/OpenCensus.Abstractions/Trace/Propagation/ITextFormat.cs

```diff
@@ -30,7 +30,7 @@ public interface ITextFormat
         ///   * allow pre-allocation of fields, especially in systems like gRPC Metadata
         ///   * allow a single-pass over an iterator (ex OpenTracing has no getter in TextMap).
         /// </summary>
-        IList<string> Fields { get; }
+        ISet<string> Fields { get; }
```

Comment:
did we get it from Java? I don't get an idea behind it.

Reply:
I just trusted the comment I saw from Java. I assume there is some requirement to pre-allocate fields in project metadata